CHANGELOG.md

```diff
@@ -6,9 +6,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- portable `download_zips` function for downloading raw data from USDA
+
+- download functionality for 2022 files
+
+### Fixed
+
+- CLI download utility can handle specific years ([#20](https://github.com/stactools-packages/usda-cdl/issues/20))
```

Comment:
```suggestion
### Added

-`download_zips` function for downloading raw data from USDA ([#22](https://github.com/stactools-packages/usda-cdl/pull/22))
- Download functionality for 2022 files ([#22](https://github.com/stactools-packages/usda-cdl/pull/22))

### Fixed

- CLI download utility can handle specific years ([#20](https://github.com/stactools-packages/usda-cdl/pull/22))
```

Normally I'd fix this up myself but it doesn't look like I have write access to your branch.